tests/test_matrix_utils.py

```
@@ -6450,6 +6451,50 @@ def test_normalize_bot_user_id_ipv6_and_ports(self, mock_logger):
         result3 = _normalize_bot_user_id("[::1]:8448", "carol")
         self.assertEqual(result3, "@carol:[::1]")
 
+    @patch("mmrelay.matrix_utils.logger")
+    def test_extract_localpart_from_mxid(self, mock_logger):
```

Ruff (0.14.13)

Unused method argument: `mock_logger`

(ARG002)